### PR TITLE
Site settings for meeting time and location

### PR DESCRIPTION
**Pull request by jnewbery**
The time and location appear in two places in the home page. Make them site settings so if we need to change them in future, we only have to change one place.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: home
 ### A weekly review club for Bitcoin Core PRs
 
 <span class="question">What is this?</span> &nbsp;A weekly club for reviewing
-Bitcoin Core PRs at **18:00 UTC on Wednesdays** on IRC.
+Bitcoin Core PRs at **{{ site.meeting_time }}** on IRC.
```
Comment (jonatack) [inline index.md]: Hacky but perhaps `Bitcoin Core PRs at **{{ site.meeting_time }}s** on IRC.` to keep "Wednesdays".

Comment (jonatack) [reply index.md]: (or define a `general_meeting_time` and a `meeting_time` in the site settings)

File: _config.yml [hunk 2]
```diff
@@ -8,7 +8,9 @@ sass:
...
-# Customise atom feed settings (this is where Jekyll-Feed gets configuration information)
-title:		           "Bitcoin Core PR Review Club"
-description:	       "A weekly review club for Bitcoin Core PRs"
-author:              "John Newbery"
+# Site settings
+title:           "Bitcoin Core PR Review Club"
+description:     "A weekly review club for Bitcoin Core PRs"
+author:          "Bitcoin Core PR Review Club contributors"
+meeting_time:    "18:00 UTC on Wednesday"
+irc_location:    "#bitcoin-core-pr-reviews IRC channel on [freenode](https://webchat.freenode.net/)"
```
Comment (pinheadmz) [inline _config.yml]: I could be wrong about this but I think markdown is only parsed/rendered in `{{content}}` sections? So for example the link will not look correct if I stick this string in the top of the page in https://github.com/bitcoin-core-review-club/bitcoin-core-review-club.github.io/pull/88

Comment (jonatack) [reply _config.yml]: `{{content}}` works but in an .html page, markdown does not... run `make preview` and open `http://localhost:4000/16702.html` to test... but you can use CSS/HTML markup if needed. I don't think you need any here though.

Comment (jnewbery) [reply _config.yml]: Ah yes. Thanks. I've replaced this with html and it seems to work.

File: index.md (hunk 1 above)
Comment (jnewbery) [reply index.md]: I'm ok with losing the s :)

**Comment by jnewbery** [comment]
Changed the site settings to `meeting_time`, `meeting_day` and `meeting_location`. That means they should be more easily usable in #88

File: index.md (hunk 1 above)
Comment (jnewbery) [reply index.md]: I've added the s back in!

**Comment by pinheadmz** [comment]
ACK @ 76fd82c7cbe49b0f0600b2f0dd43b3dd109cd878 update to #88 incoming.

**Comment by jnewbery** [comment]
Merged in #88

**Pull request closed without merging**